app/views/projects/show.html.erb

```diff
@@ -9,19 +9,19 @@
     <form method="GET">
       <input id="project-tagging-content-item-search"
              type="text"
-             class="form-control js-bulk-tagging-search-filter"
-             name="query"
-             value="<%= bulk_search.query %>"
+             class="form-control"
+             name="title_search"
+             value="<%= query.params[:title_search] %>"
              placeholder="Filter by title...">
 
-      <% Projects::BulkSearch::TAGGED_STATES.each do |state|  %>
+      <% ProjectFilterQuery::FILTERS.each do |state|  %>
```

Comment:
could this be an instance method?

Reply:
Yep, I've updated this in https://github.com/alphagov/content-tagger/pull/465/commits/7fd7ddd23f10c1ae1059fc4c76be10ee274a3dec#diff-2476f44a5f0802d146b181eeb56e8a12R17